client/src/containers/profile/user-details/index.tsx

```diff
@@ -4,13 +4,21 @@ import AccountDetails from "@/containers/profile/account-details";
 import EditPassword from "@/containers/profile/edit-password";
 import UpdateEmailForm from "@/containers/profile/update-email";
 
+import { Card } from "@/components/ui/card";
+
 const UserDetails: FC = () => {
   return (
-    <>
-      <AccountDetails />
-      <UpdateEmailForm />
-      <EditPassword />
-    </>
+    <div className="mt-4 space-y-4">
+      <Card variant="secondary" className="border-dashed p-4">
```

Comment:
I think all `Card` components have a common padding, I would move this padding to its common styles.